consultation_analyser/consultations/views/filters.py

```diff
@@ -0,0 +1,34 @@
+from django.db.models import Count
+from django.http import HttpRequest
+
+from .. import models
+
+
+def get_applied_filters(request: HttpRequest):
+    return {
+        "keyword": request.GET.get("keyword", ""),
+        "theme": request.GET.get("theme", "All"),
+        "opinion": request.GET.get("opinion", "All"),
+    }
+
+
+def get_filtered_responses(question: models.Question, applied_filters):
+    queryset = models.Answer.objects.filter(question=question, free_text__icontains=applied_filters["keyword"])
+    if applied_filters["theme"] != "All" and applied_filters["theme"] != "No theme":
```

Comment:
Are we actually using this "No theme"? Can probs be removed. I think outliers will be considered in a different PR?

Reply:
Yeah, good point. I'll remove it.